dockerfiles/igv/makeigvpesr_trio.py

```diff
@@ -60,7 +60,7 @@ def cram_info_readin(cram_file):
         if not pin[0] in out.keys():
             out[pin[0]] = pin[1:]
     fin.close()
-    return(out)
+    return (out)
```

Comment:
Parenthesis have no effect
```suggestion
    return out
```